pyproject.toml

```diff
@@ -111,7 +111,7 @@ inspect = "inspect_ai._cli.main:main"
 
 [project.optional-dependencies]
 inspect_evals = [
-    "datasets",
+    "datasets>=2.21", # This is the last version of HuggingFace datasets befor the v3.0.0 release. Needed for a working datasets.load_dataset call for several of the evals.
```

Comment:
We should probably also upper bound then (e.g. `datasets>=2.21,<3`). But really we should very soon get our stuff working with datasets 3 and then require that.

Reply:
To be clear, I'm pretty sure our stuff works with datasets 3! I just don't fully know if they've done any breaking changes, and didn't want to pin people. If you think its fine, can just bump up to 3.

Reply:
If as far as we know we work with both 2 and 3 then we should just leave it along (no bounding) and then apply constraints/workarounds as required when we discover issues.

Reply:
So in particular, our stuff doesn't work with versions of datasets below 2.21.

Reply:
okay, that sounds great then!